x-pack/plugins/siem/public/components/page/hosts/authentications_table/index.tsx

```diff
@@ -303,3 +305,24 @@ const getAuthenticationColumns = (): [
       }),
   },
 ];
+
+const getAuthenticationColumnsCurated = (pageType: hostsModel.HostsType) => {
+  const columnsCurated = getAuthenticationColumns(pageType);
+
+  if (pageType === 'details') {
```

Comment:
this `if` can be simplified:
```
  if (pageType === 'details') {
    // Array of column names to exclude from host details pages
    return [i18n.LAST_FAILED_DESTINATION, i18n.LAST_SUCCESSFUL_DESTINATION].reduce((acc, name) => {
      acc.splice(acc.findIndex(column => column.name === name), 1);
      return acc;
    }, columnsCurated);
  }
```

We also need to be adding a test for any `if` that we add

Reply:
Awesome; thanks!